packages/ibm-products/src/components/Datagrid/Datagrid.stories.jsx

```diff
@@ -56,6 +57,27 @@ export default {
     },
   },
   excludeStories: ['getBatchActions'],
+  decorators: [
+    (story) => (
+      <div>
+        <Annotation
+          type="deprecation-notice"
+          text={
+            <div>
+              This component is deprecated and will be removed in the next major
```

Comment:
Found one more spot to use the updating wording from @elycheea

Reply:
Or maybe just a slight update, since we'll be keeping it in the next major

Reply:
updated it to match the other wording

Reply:
Haha, I removed the “major” language. I expect it to still be around in v3 but remain deprecated, but if we do see usage drop in telemetry, that would change things. ;)

Reply:
Yep, that aligns with the changes David made [here](https://github.com/carbon-design-system/ibm-products/pull/6419/commits/f37e48a11d07a6f34cd65b6fa7c1d0409a2dfe62)